test/javascripts/helpers/qunit-helpers.js.es6

```diff
@@ -128,6 +128,16 @@ export function acceptance(name, options) {
       resetPostCookedDecorators();
       resetOneboxCache();
       resetCustomPostMessageCallbacks();
+
+      Discourse._runInitializer("instanceInitializers", function(
+        initializerName,
+        initializer
+      ) {
+        if (initializer && initializer.teardown) {
+          initializer.teardown(Discourse.__container__);
```

Comment:
Is this working? Because I did a test locally and found that Discourse.__container__ here was not the same container that the initializer was run with.